samples/gtk_accel_group.cr

```diff
@@ -0,0 +1,16 @@
+require "gobject/gtk/autorun"
+
+window = Gtk::ApplicationWindow.new(title: "AccelGroup", border_width: 20)
+window.connect "destroy", ->Gtk.main_quit
+window.add Gtk::Label.new("Press Ctrl+S")
+
+accel_group = Gtk::AccelGroup.new
+# Passing a block
+closure = GObject::Closure.new do
+  puts "Ctrl+S pressed!"
+end
+
+accel_group.connect(Gdk::KEY_S, Gdk::ModifierType::CONTROL_MASK, Gtk::AccelFlags::ZERO_NONE, closure)
```

Comment:
```suggestion
accel_group.connect(Gdk::KEY_S, :control_mask, :zero_none, closure)
```

I didn't test this but given Crystal's symbol to enum conversion I think that should work :)

Reply:
It worked, nice. I didn't know about this language feature. I'll update the PR.